### PR TITLE
Add new wait event types introduced in PG13 and PG14

### PR DESCRIPTION
**Pull request by keiko713**
This adds new wait event types that were introduced PG13 and 14. There were lots of renaming happened with these 2 versions.
For the renames, I took the following approach:

* add a new name in the comment of `compact_activity_snapshot.proto`, following by the comma
* change the enum key to the most recent postgres version one
* add a new name as a new row of  mappings/wait_event.json, used the same number as a previous name one for the mapping

I would recommend that you review with the each commit, I added the link to the corresponding Postgres side change in the commit message.

Closes https://github.com/pganalyze/pganalyze/issues/1302

### CODE REVIEW

File: compact_activity_snapshot.proto [hunk 1]
```diff
@@ -48,12 +48,12 @@ message Backend {
     WAIT_EVENT_LWLOCK_WAL_BUF_MAPPING_LOCK = 107; // WALBufMappingLock
     WAIT_EVENT_LWLOCK_WAL_WRITE_LOCK = 108; // WALWriteLock
     WAIT_EVENT_LWLOCK_CONTROL_FILE_LOCK = 109; // ControlFileLock
-    WAIT_EVENT_LWLOCK_CHECKPOINT_LOCK = 110; // CheckpointLock
-    WAIT_EVENT_LWLOCK_C_LOG_CONTROL_LOCK = 111; // CLogControlLock
-    WAIT_EVENT_LWLOCK_SUBTRANS_CONTROL_LOCK = 112; // SubtransControlLock
+    WAIT_EVENT_LWLOCK_CHECKPOINT_LOCK = 110; // CheckpointLock (removed in 14)
+    WAIT_EVENT_LWLOCK_C_LOG_CONTROL_LOCK = 111; // CLogControlLock, XactSLRULock
```
Comment (keiko713) [inline compact_activity_snapshot.proto]: @lfittl I forgot if we decided to update the name here too like the following or not. Do you... remember?

```suggestion
    WAIT_EVENT_LWLOCK_XACT_SLRU_LOCK = 111; // CLogControlLock, XactSLRULock
```

Comment (lfittl) [reply compact_activity_snapshot.proto]: I think we said we'd update the enum key name it to what its now named, i.e. in the most recent Postgres version. I think that'd be `WAIT_EVENT_LWLOCK_XACT_SLRU_LOCK` in this case.

Comment (keiko713) [reply compact_activity_snapshot.proto]: cool cool 👍  I'll make a new commit to update all the things 🎉

File: compact_activity_snapshot.proto [hunk 2]
```diff
@@ -262,7 +284,7 @@ message Backend {
     WAIT_EVENT_WAL_SYNC = 965; // WALSync
     WAIT_EVENT_WAL_SYNC_METHOD_ASSIGN = 966; // WALSyncMethodAssign
     WAIT_EVENT_WAL_WRITE = 967; // WALWrite
-    WAIT_EVENT_PROC_SIGNAL_BARRIER = 968; // ProcSignalBarrier
+    WAIT_EVENT_PROC_SIGNAL_BARRIER = 968; // ProcSignalBarrier (moved to IPC category in 13)
```
Comment (keiko713) [inline compact_activity_snapshot.proto:287]: [note for the future] in Postgres 14, 4 new pgstat_get_wait_io events were added. However, this ones were never used and planned to be removed in Postgres 15, therefore I didn't add them here. See below commits for the context:

https://github.com/postgres/postgres/commit/464824323e57dc4b397e8b05854d779908b55304
https://github.com/postgres/postgres/commit/671eb8f34404d24c8f16ae40e94becb38afd93bb
https://github.com/postgres/postgres/commit/a6a0ae127e76da58e0396cd0f21a97066da2115b

**Review by lfittl — APPROVED** [review]
:+1: Thanks for going through this in detail!

**Pull request merged**